### ISSUE TITLE
Cells in GameGrid are not changing appearances after being clicked.

### ISSUE THREAD

**Issue by RobRoseKnows** [RobRoseKnows/nonogram#2]
As of 25c3b11729594e0ea113cceca028e568ab4bde8c.

Problem likely in GameScreen.java on GameScreen.java:48 where postInvalidate(); does not properly refresh the views.
When looking at the LogChat readout we can see that getView() was not called since it was originally rendered. I clicked several cells to see if it would rerender, it did not.

> 05-08 17:15:29.910: D/dalvikvm(1794): GC_FOR_ALLOC freed 87K, 5% free 3033K/3184K, paused 38ms, total 50ms
> 05-08 17:15:29.910: I/dalvikvm-heap(1794): Grow heap (frag case) to 3.499MB for 500416-byte allocation
> 05-08 17:15:29.960: D/dalvikvm(1794): GC_FOR_ALLOC freed 7K, 5% free 3514K/3676K, paused 47ms, total 47ms
> 05-08 17:15:30.040: D/Cell Constructor(1794): Cell created (0, 0) = true
> 05-08 17:15:30.040: D/Cell Constructor(1794): Cell created (0, 1) = true
> 05-08 17:15:30.040: D/Cell Constructor(1794): Cell created (0, 2) = true
> 05-08 17:15:30.050: D/Cell Constructor(1794): Cell created (0, 3) = true
> 05-08 17:15:30.050: D/Cell Constructor(1794): Cell created (0, 4) = false
> 05-08 17:15:30.050: D/Cell Constructor(1794): Cell created (1, 0) = false
> 05-08 17:15:30.050: D/Cell Constructor(1794): Cell created (1, 1) = true
> 05-08 17:15:30.050: D/Cell Constructor(1794): Cell created (1, 2) = true
> 05-08 17:15:30.050: D/Cell Constructor(1794): Cell created (1, 3) = false
> 05-08 17:15:30.050: D/Cell Constructor(1794): Cell created (1, 4) = false
> 05-08 17:15:30.050: D/Cell Constructor(1794): Cell created (2, 0) = true
> 05-08 17:15:30.050: D/Cell Constructor(1794): Cell created (2, 1) = true
> 05-08 17:15:30.050: D/Cell Constructor(1794): Cell created (2, 2) = false
> 05-08 17:15:30.050: D/Cell Constructor(1794): Cell created (2, 3) = false
> 05-08 17:15:30.050: D/Cell Constructor(1794): Cell created (2, 4) = false
> 05-08 17:15:30.050: D/Cell Constructor(1794): Cell created (3, 0) = false
> 05-08 17:15:30.050: D/Cell Constructor(1794): Cell created (3, 1) = true
> 05-08 17:15:30.070: D/Cell Constructor(1794): Cell created (3, 2) = true
> 05-08 17:15:30.070: D/Cell Constructor(1794): Cell created (3, 3) = true
> 05-08 17:15:30.070: D/Cell Constructor(1794): Cell created (3, 4) = true
> 05-08 17:15:30.070: D/Cell Constructor(1794): Cell created (4, 0) = false
> 05-08 17:15:30.070: D/Cell Constructor(1794): Cell created (4, 1) = true
> 05-08 17:15:30.070: D/Cell Constructor(1794): Cell created (4, 2) = true
> 05-08 17:15:30.070: D/Cell Constructor(1794): Cell created (4, 3) = false
> 05-08 17:15:30.090: D/Cell Constructor(1794): Cell created (4, 4) = true
> 05-08 17:15:30.100: D/GAME_GRID_ADAPTER(1794): 0 true, 0 true, 0 true, 0 true, 0 false, 0 false, 0 true, 0 true, 0 false, 0 false, 0 true, 0 true, 0 false, 0 false, 0 false, 0 false, 0 true, 0 true, 0 true, 0 true, 0 false, 0 true, 0 true, 0 false, 0 true, 
> 05-08 17:15:30.160: I/GET_VIEW(1794): getView called position: 0
> 05-08 17:15:30.250: W/EGL_emulation(1794): eglSurfaceAttrib not implemented
> 05-08 17:15:30.250: I/GET_VIEW(1794): getView called position: 1
> 05-08 17:15:30.270: I/GET_VIEW(1794): getView called position: 2
> 05-08 17:15:30.270: I/GET_VIEW(1794): getView called position: 3
> 05-08 17:15:30.270: I/GET_VIEW(1794): getView called position: 4
> 05-08 17:15:30.280: I/GET_VIEW(1794): getView called position: 5
> 05-08 17:15:30.290: I/GET_VIEW(1794): getView called position: 6
> 05-08 17:15:30.300: I/GET_VIEW(1794): getView called position: 7
> 05-08 17:15:30.300: I/GET_VIEW(1794): getView called position: 8
> 05-08 17:15:30.310: I/GET_VIEW(1794): getView called position: 9
> 05-08 17:15:30.310: I/GET_VIEW(1794): getView called position: 10
> 05-08 17:15:30.310: I/GET_VIEW(1794): getView called position: 11
> 05-08 17:15:30.330: I/GET_VIEW(1794): getView called position: 12
> 05-08 17:15:30.330: I/GET_VIEW(1794): getView called position: 13
> 05-08 17:15:30.340: I/GET_VIEW(1794): getView called position: 14
> 05-08 17:15:30.340: I/GET_VIEW(1794): getView called position: 15
> 05-08 17:15:30.340: I/GET_VIEW(1794): getView called position: 16
> 05-08 17:15:30.350: I/GET_VIEW(1794): getView called position: 17
> 05-08 17:15:30.350: I/GET_VIEW(1794): getView called position: 18
> 05-08 17:15:30.370: I/GET_VIEW(1794): getView called position: 19
> 05-08 17:15:30.370: I/GET_VIEW(1794): getView called position: 20
> 05-08 17:15:30.380: I/GET_VIEW(1794): getView called position: 21
> 05-08 17:15:30.380: I/GET_VIEW(1794): getView called position: 22
> 05-08 17:15:30.380: I/GET_VIEW(1794): getView called position: 23
> 05-08 17:15:30.400: I/GET_VIEW(1794): getView called position: 24
> 05-08 17:15:30.410: I/GET_VIEW(1794): getView called position: 0

**Comment by RobRoseKnows** [comment]
I've found a few StackOverflow answers possibly related.

http://stackoverflow.com/questions/5991968/how-to-force-an-entire-layout-view-refresh
This suggests a wide variety of things, some of which I've tried already (in particular postInvalidate() and invalidate()). Although it seems like that function is intended for this purpose, it doesn't seem to work very well. There are several alternatives provided, some of them look potentially irritating to implement.
